### DISCUSSION TITLE
Deepfake 深偽技術的技術濫用與道德困境，大眾正要開始面對

### DISCUSSION THREAD

**Discussion by 110409538** [1101-GS3073-C/Python-Community-GS3073-C#78] [科技新知]
https://technews.tw/2021/10/23/we-are-facing-the-abusing-of-technology-and-moral-dilemma/

![image](https://user-images.githubusercontent.com/90551832/138579101-73dc5f7b-801b-4aab-8733-8490651eb7f5.png)
▲ 左邊是真人塔金，右邊是電腦動畫做的塔金。

科技本就是一把雙刃劍.當用在對的地方,可以讓生活變得更方便.如果錯誤的使用則會造成不好的影響和社會問題等.現在的年代,網絡上充斥著許多訊息,人們必須自己辨別真假.但是,有多少人是有足夠的能力能夠辨識的.

**Comment by boson13579** [comment]
deepfake 的出現證明了人工智慧的威力，我覺得，在這資訊爆炸的時代，我們不可能希望所有獲得的資訊都是正確的，所以我們必須抱持自我的價值觀，持續了解新的技術，才能不被假資訊帶著走。

**Comment by larry40305** [comment]
某Youtuber讓這個議題在台灣受到熱烈討論。台灣應該已經在開始擬定相關法律了。
